server/tests/auth/test_enforcer.py

```
@@ -0,0 +1,205 @@
+import pytest
+
+from utils.auth import enforcer as enforcer_module
+from utils.auth.enforcer import assign_role_to_user
+
+
+# ---------------------------------------------------------------------------
+# Fixtures
+# ---------------------------------------------------------------------------
+
+
+@pytest.fixture()
```

Ruff (0.15.12)

[warning] Use `@pytest.fixture` over `@pytest.fixture()`

Remove parentheses

(PT001)